src/encoded/schemas/genetic_modification.json

```diff
@@ -986,6 +1017,15 @@
                 "homologous recombination"
             ]
         },
+        "MOI": {
+            "title": "Multiplicity of infection",
+            "description": "The ratio or frequency of viral particles introduced to cells.",
```

Comment:
Should there be something that indicates the meaning of high and low.  The description says that it is a ratio, but we are not providing a ratio but rather a classification of that ratio.

Reply:
This conversation is on the jira ticket